api/src/domain/types.ts

```diff
@@ -33,6 +33,16 @@ export interface UserDataClient {
   getNeedTaxIdEncryptionUsers: () => Promise<UserData[]>;
 }
 
+export interface BusinessesDataClient {
+  get: (businessId: string) => Promise<Business>;
+  put: (businessData: Business) => Promise<Business>;
+  deleteBusiness: (businessId: string) => Promise<void>;
+  findByBusinessName: (businessName: string) => Promise<Business | undefined>;
+  findByNAICSCode: (naicsCode: string) => Promise<Business[]>;
+  findByIndustry: (industry: string) => Promise<Business[]>;
```

Comment:
[dust/pebble] `findAllByIndustry` to indicate that it's returning multiple businesses